manifest.json.template

```diff
@@ -64,13 +64,16 @@
       "explode": "true"
     },
     "org.zowe.explorer-jes": {
-      "version": "~1.0.0-STAGING"
+      "version": "~1.0.3-PR-207"
```

Comment:
Need to revert version without PR-207 before merging.

Reply:
reverted